package.json

```diff
@@ -20,7 +20,9 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "react-bootstrap": "^0.31.5",
-    "react-fontawesome": "^1.6.1"
+    "react-c3js": "^0.1.20",
+    "react-fontawesome": "^1.6.1",
+    "patternfly": "^3.28.0"
```

Comment:
I'm OK with moving `patternfly` as a hard dependency if we remove it from `devDependencies` and `peerDependencies`. This will enforce a bit more protection at the cost of some flexibility downstream it seems... 

Please make sure to regenerate package-lock after doing `rm -rf node_modules` and re-running `npm install` w/ your package.json updated. 😸

Reply:
:+1: removed from `devDependencies` and `peerDependencies`

Reply:
whats the motivation for this change? I would assume peer dependency allow projects not to break more often? :)

Reply:
@ohadlevy 

a. reverted to unchanged package.json.

Using the package.json without depending on patternfly didn't work for me before with strange errors about less ?@#! , but it was propably something wrong in my set up.

On clean `git clone` it started to work again ... so I reverted to old package.json and it works now.

cc @sharvit @ohadlevy 

@sharvit - thanks for helping debug that.